doc/README_Generic.md

```diff
@@ -0,0 +1,61 @@
+Generic YaST README File
+========================
+
+YaST Development Documentation
+------------------------------
+
+- [YaST development documentation](http://yast.github.io/documentation.html)
+- [Contribution Guidelines](http://yast.github.io/guidelines.html)
+- [YaST architecture](http://yastgithubio.readthedocs.org/en/latest/architecture)
+- [Development environment](https://en.opensuse.org/openSUSE:YaST:_Preparing_the_Development_Environment)
```

Comment:
Shouldn't we actually merge it from Wiki to a generic README file? I don't like having it at two places :-/ as that would only cause confusion. The wiki page could link here